mint.json

```diff
@@ -443,13 +445,22 @@
             "processing/time-travel-queries"
           ]
         },
+        {
+          "group": "Serve results",
```

Comment:
Not sure if we need to follow the other three items 👀 "Serve data"?

Reply:
When we have both "Serve data" and "Deliver data", the distinction between these two are not alway clear. Maybe just "Query data"?